src/hw04_task02_cities.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
what about Brest in Belarus and France?

Reply:
Conclusion changed to city-country. Now it’s clear where the cities are found. Duplicate names are displayed several times, but the country is indicated.

Reply:
changed list names